prime-router/src/main/kotlin/cli/tests/HistoryApiTest.kt

```diff
@@ -71,28 +69,9 @@ class HistoryApiTest : CoolTest() {
     override val description = "Test the History/Lineage API"
     override val status = TestStatus.SMOKE
 
-    // todo this code was copied from oktaAccessToken in SettingCommands.kt
```

Comment:
Does any of this still apply to the copy now in AuthTests?

Reply:
Not really. There was a suggestion on how to reformat, I took a different road.